apps/backend/src/libs/modules/bcrypt/base-bcrypt.module.ts

```diff
@@ -0,0 +1,20 @@
+import * as bcrypt from "bcrypt";
+
+class BaseBcrypt {
+	async hash(password: string): Promise<{ hash: string; salt: string }> {
+		const salt = await bcrypt.genSalt();
+		const hash = await bcrypt.hash(password, salt);
+
+		return { hash, salt };
+	}
+
+	async verify(
+		plainPassword: string,
+		hashedPassword: string,
+	): Promise<boolean> {
+		const match = await bcrypt.compare(plainPassword, hashedPassword);
+		return match;
+	}
```

Comment:
There is no need to create an extra variable
```suggestion
	verify(
		plainPassword: string,
		hashedPassword: string,
	): Promise<boolean> {
		return bcrypt.compare(plainPassword, hashedPassword);
	}
```